src/scripts/components/Play.js

```diff
@@ -8,12 +8,13 @@ import VideoRecord from 'records/VideoRecords'
 import VideoOverlay from 'components/VideoOverlay'
 
 import type { RouteMatch } from 'types/ApplicationTypes'
+import Debug from 'containers/DebugContainer'
```

Comment:
Should this be removed before merging?

Reply:
we are in full dev mode, and I find it very useful. We should remove it once we go public, obviously.